README.md

```diff
@@ -735,3 +793,26 @@ $lastUsers = $users->getLastPage();
 // Load a specific page
 $otherUsers = $users->getPage(12);
 ```
+
+## Testing
+
+The SDK comes with a handy `mock` method on the `ApiClient` class. To use this, pass in the name of the endpoint you want to mock. You'll get a `\Mockery\MockInterface` object back. Once you set the mock, any subsequent calls to that endpoint will return the mocked object.
+
+```php
+// From within the tests/ApiClientTest.php file...
+public function testMockReturnsProperMock()
+{
+    $client = ApiClientFactory::createClient();
+    $mockedWorkflows = $client->mock('workflows');
+
+    $this->assertInstanceOf(WorkflowsEndpoint::class, $mockedWorkflows);
+    $this->assertInstanceOf(MockInterface::class, $mockedWorkflows);
+
+    $this->assertSame(
+        $mockedWorkflows,
+        $client->workflows()
+    );
+}
+``` 
```

Comment:
What do you think about replacing this with an example of how the mock would be used within a phpunit test?

Reply:
that is how it would be used...